WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StorePerformanceViewModel.swift

```diff
@@ -160,8 +160,9 @@ final class StorePerformanceViewModel: ObservableObject {
         // Preemptively show any cached content
         periodViewModel?.loadCachedContent()
 
-        // Stop if data is relatively new
-        if !forceRefresh && DashboardTimestampStore.isTimestampFresh(for: .performance, at: timeRange.timestampRange) {
+        // Stop if data is relatively new (unless forceRefresh or in screenshot mode)
+        if !forceRefresh && !ProcessConfiguration.shouldRespectScreenshotMode &&
+           DashboardTimestampStore.isTimestampFresh(for: .performance, at: timeRange.timestampRange) {
```

Comment:
So the issue encountered here is that since screenshots are run multiple times at a close time, subsequent runs will make the `isTimestampFresh()` return true and skip the mock syncing, making the card return empty values:
![Screenshot 2024-11-15 at 16 20 00](https://github.com/user-attachments/assets/cddb4dfe-3b49-4a8e-81cd-3fac882d68ca)

The `shouldRespectScreenshotMode` was a hack to make it do mock syncing all the time. 

The code before this part actually tries to get mocked content, but somehow it's not returning anything.

```
        // Preemptively show any cached content
        periodViewModel?.loadCachedContent()
  ```

I'll try to remove this hack and figure out why the cached content are not found during multiple runs.

Reply:
Found the bug, I'll make a separate PR for it as it also affects the app.

Reply:
https://github.com/woocommerce/woocommerce-ios/issues/14436

Reply:
PR to fix: https://github.com/woocommerce/woocommerce-ios/pull/14438

Reply:
This is now solved in https://github.com/woocommerce/woocommerce-ios/pull/14438

Reply:
screenshot mode flag is removed in https://github.com/woocommerce/woocommerce-ios/pull/14335/commits/c64a1d3111c045e85a9b4f64268f473b8cf9f192